test/plugin_functional/test_suites/core_plugins/rendering.ts

```diff
@@ -293,6 +293,7 @@ export default function ({ getService }: PluginFunctionalProviderContext) {
         'xpack.index_management.editableIndexSettings (any)',
         'xpack.index_management.enableDataStreamsStorageColumn (any)',
         'xpack.index_management.enableMappingsSourceFieldSection (any)',
+        'xpack.index_management.enableSemanticText (any)',
```

Comment:
Quesiton: Is there a reason this is showing up `(any)` instead of `(boolean)`?

If it has to be `(any)` would you please add a comment stating that it will only contain boolean values?

Reply:
No real reason other than that I copy-pasted the lines above, I'd be happy to update this to `boolean`